### PR TITLE
Add Notifiers API

### PR DESCRIPTION
**Pull request by EtienneM**
Related to Scalingo/dashboard-vue#252

### CODE REVIEW

File: src/Notifiers/index.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      this._client.apiClient().get(`/apps/${appId}/notifiers`),
+      'notifiers',
+    )
+  }
+
+  /**
+   * Add a notifier to an application
+   * @see https://developers.scalingo.com/notifiers#add-a-notifier
+   * @param {String} appId ID of the app
+   * @param {Notifier} notifier New notifier configuration
```
Comment (johnsudaar) [inline src/Notifiers/index.js]: It would be nice to use another object here (to show the optionnal params).

File: src/Notifiers/index.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      'notifier',
+    )
+  }
+
+  /**
+   * Upgrade a notifier
+   * @see https://developers.scalingo.com/notifiers#update-a-notifier
+   * @param {String} appId ID of the current application
+   * @param {String} notifierId ID of the current notifier
+   * @param {Notifier} notifier Updated notifier configuration
```
Comment (johnsudaar) [inline src/Notifiers/index.js]: Same thing here.

File: src/Notifiers/index.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+   * Send a test notification to the notifier
+   * @param {String} appId The ID of the current application
+   * @param {String} notifierId The ID of the notifier
+   * @return {Promise<? | APIError>}
+   */
+  test(appId, notifierId) {
+    return unpackData(
+      this._client
+        .apiClient()
+        .get(`/apps/${appId}/notifiers/${notifierId}/test`),
```
Comment (johnsudaar) [inline src/Notifiers/index.js]: This is a POST.

File: src/Notifiers/index.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+   * @return {Promise<? | APIError>}
+   */
+  destroy(appId, notifierId) {
+    return unpackData(
+      this._client.apiClient().delete(`/apps/${appId}/notifiers/${notifierId}`),
+    )
+  }
+
+  /**
+   * Send a test notification to the notifier
```
Comment (johnsudaar) [inline src/Notifiers/index.js]: Missing doc link.

File: .travis.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ before_deploy:
 install: yarn
 cache: yarn
 node_js:
-- 11.6
+- 10.16
```
[the remark this answers is not in the document]
Comment (EtienneM) [reply .travis.yml]: The eslint dependency needs either a 10.x or a 12.x. It failed in TravisCI. I chose to downgrade to use the latest LTS version.

File: package.json [hunk 6]
```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "rollup -c",
     "docs": "esdoc",
+    "lint": "eslint --ext .js ./src ./test",
```
[the remark this answers is not in the document]
Comment (EtienneM) [reply package.json]: Yes I just need to find out how.... :sweat_smile: 
I started digging and didn't find anything

File: src/Notifiers/index.js (hunk 3 above)
Comment (EtienneM) [reply src/Notifiers/index.js]: Thanks

File: src/Notifiers/index.js (hunk 1 above)
Comment (EtienneM) [reply src/Notifiers/index.js]: OK I was a bit lazy but you're right ^^

**Pull request merged**